README.md

```
@@ -92,12 +92,103 @@ Before getting started, ensure you have:
+   - RX → TX
+4. Put device in flash mode:
+   - Connect the BOOT pin to GND using a jumper wire
+   - Power up the device (keeping BOOT connected to GND)
+   - Remove the BOOT to GND connection
+5. Visit [ESPHome Web](https://web.esphome.io)
+6. Connect to your device and flash the firmware
+7. After successful flash, device will restart and be ready for OTA updates
+
+#### Detailed Visual Guides
+For step-by-step visual instructions, you can reference these existing guides:
+- 🇬🇧 [WirelessThings Guide](https://wirelessthings.net/how-to-flash-sonoff-tx-ultimate-with-esphome) -
```

Markdownlint (0.37.0)

Expected: 80; Actual: 104
Line length

(MD013, line-length)